README.md

```diff
@@ -6,6 +6,41 @@
 
 Feign is a Java to HTTP client binder inspired by [Retrofit](https://github.com/square/retrofit), [JAXRS-2.0](https://jax-rs-spec.java.net/nonav/2.0/apidocs/index.html), and [WebSocket](http://www.oracle.com/technetwork/articles/java/jsr356-1937161.html).  Feign's first goal was reducing the complexity of binding [Denominator](https://github.com/Netflix/Denominator) uniformly to HTTP APIs regardless of [ReSTfulness](http://www.slideshare.net/adrianfcole/99problems).
 
+---
+# Roadmap
+## Feign 11 and beyond 
```

Comment:
Do you think work mention that we will be breaking feign internal API? Which may affect people

Reply:
Also, should we have a compromise of what we are not going to break?

Reply:
I fat finger the resolve conversation button.


Anyway, probably the retry changes will break some API too.